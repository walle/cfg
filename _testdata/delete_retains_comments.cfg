# Test data to verify that new values can be deleted programmatically

  # Foo value (string)
foo = bar

# Bar value (string)
bar = foo

# Foobar value (string)
  foobar = baz
