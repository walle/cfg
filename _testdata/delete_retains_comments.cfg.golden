# Test data to verify that new values can be deleted programmatically

  # Foo value (string)
foo = bar

# Bar value (string)

# Foobar value (string)
  foobar = baz
